pydm/widgets/baseplot.py

```diff
@@ -66,6 +64,8 @@ def __init__(self, color=None, lineStyle=None, lineWidth=None, **kws):
         self.setSymbolBrush(None)
         if color is not None:
             self.color = color
+        if kws.get("name", None):
```

Comment:
This is needed because the name attribute is set at the constructor (__init__). PlotDataItem sets the name attribute with a call to the setData() method.

Reply:
Sorry to insist on the subject @hmbui, but the `setData` method is called at the constructor of `PlotDataItem`:

https://github.com/pyqtgraph/pyqtgraph/blob/a690ea41a0d236d65ba20ad2b0ae8b704d335afb/pyqtgraph/graphicsItems/PlotDataItem.py#L177

This means that at this point the name of curve was already set, right?

Reply:
You're right, I overlooked that. Thank you for pointing that out.